packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -60,7 +60,8 @@ export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
         inlineLabel && {
           order: 1,
           marginLeft: 16
-        }
+        },
+      inlineLabel && { overflow: 'hidden' }
```

Comment:
shouldn't this be word-break instead? is there a reason to use overflow instead?

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
Yeah, that seems to work too. I'll switch it out.